lighthouse-core/report/html/renderer/report-ui-features.js

```diff
@@ -744,6 +747,21 @@ class DropDown {
     }
   }
 
+  /**
+   * Focus out handler for the drop down menu.
+   * @param {Event} e
+   */
+  onMenuFocusOut(e) {
+    // The focusout event is not supported in our current version of typescript (3.5.3)
+    // https://github.com/microsoft/TypeScript/issues/30716
+    const focusEvent = /** @type {FocusEvent} */ (e);
+    const focusedEl = /** @type {?HTMLElement} */ (focusEvent.relatedTarget);
```

Comment:
is this nullable?

Reply:
Yes, relatedTarget might be null.
https://developer.mozilla.org/en-US/docs/Web/API/FocusEvent/FocusEvent

Reply:
ah, I see, this is another thing that newer TS has corrected.